phpstan.neon

```diff
@@ -3,6 +3,9 @@ includes:
     - phpstan-params.neon
 
 parameters:
+    excludePaths:
+        - src/Mapping/Driver/LoadMappingFileImplementation.php
+        - src/Mapping/GetReflectionClassImplementation.php
```

Comment:
Why do we need to exclude them?

Reply:
Maybe when I initially tried this, PHPStan said it could not analyze them. Right now, it can, so I will change that.

Reply:
Try inverting the `if` condition so that the version of the trait _with_ native return types comes first.

Reply:
@derrabus thanks, that worked!